draft-tsai-duration-00.xml

```diff
@@ -53,21 +51,16 @@ This document defines a particular profile of ISO 8601 for durations.</t>
 
 <section anchor="design-issues" title="Design Issues">
 <section anchor="accuracy" title="Accuracy">
-<t>ISO 8601 permits the use of year, month, week, and day units to represent a duration, but does not define a context independent way to convert such larger units to smaller units such as hour, minute, and second.</t>
-<t>A minute is defined as being 60 seconds long. An hour is defined as being 60 minutes long. In contrast, a day is nominally 24 hours long, but subject to the insertion and removal of leap seconds, and the insertion and removal of intervals by local authorities.
-Consequently, the length of a day is context dependent and therefore unsuitable for an accurate representation of a duration.
-Similarly, the length of a year, month, or week is dependent on context such as what timestamp and what jurisdiction the duration is relative to.
-For example, 1 year starting relative to January 1st, 2000 is 366 days long, while 1 year starting relative to January 1st, 2001 is 365 days long, due to the existence of leap years.</t>
-<t>For interoperability, accuracy is prioritized and allowing year, month, week, and day units would violate this property.
-A formatted duration must be able to represent an exact duration in a context independent manner.</t>
-
+<t>ISO 8601 permits the use of year, month, week, and day units in representing a duration, but none of these can be reliably converted to a fixed duration value without further context. Years vary in length depending on leap years and seconds, months by definition, and days depending on leap seconds.</t>
```

Comment:
> Years vary in length depending on leap years and seconds, months by definition, and days depending on leap seconds.

This reads a little confusing to me. Perhaps, more simply as:

> Years, months, and days vary in length depending on leap years and/or leap seconds.

It's slightly less precise, but adequately conveys the problem.

Reply:
Agree.